imports/file_redolog_manager.go

```diff
@@ -87,15 +87,15 @@ func NewFileRedoLogManager(rotationInterval int64, maxRedoLogSize int64, diskSto
 }
 
 // Close closes the current log file.
-func (r *fileRedologManager) Close() {
+func (r *FileRedologManager) Close() {
```

Comment:
check the error of close.... same to other places as well, you need to check the return value of other close as well